Testscripts/Windows/VALIDATE-DA.ps1

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+    This script validates Dependency Agent.
+
+.Description
+    The script will install the Latest Dependency Agent and validate install/uninstall
+    scenarios as well as enable/disable scenarios.
+'
+#>
+param([String] $TestParams,
+      [object] $AllVmData)
+function Main {
+    param (
+        $VMName,
```

Comment:
Recommend adding data types to the function parameters.

Reply:
Done